firefox_ui_tests/functional/toolbar/manifest.ini

```diff
@@ -1,2 +1,3 @@
 [test_back_forward.py]
 [test_home_button.py]
+[test_access_location_bar.py]
```

Comment:
The test is in the current directory because that's what you requested when we asked you during the training. I will move it.